src/runtime/manual_tests/firebase-tests.ts

```diff
@@ -58,8 +58,8 @@ describe('firebase', function() {
     return storage;
   }
 
-  after(() => {
-    storageInstances.map(s => s.shutdown());
+  after(async () => {
+    await Promise.all(storageInstances.map(s => s.shutdown()));
```

Comment:
Is this actually required? I deliberately didn't await since this function doesn't care about the result of the shutdown methods.

Reply:
I could conceive of a case where we might care about errors on shutdown.  For example if there are pending requests that are not complete.

Reply:
Ok, sounds fair.